### ISSUE TITLE
simulated beacons not detecting

### ISSUE THREAD

**Issue by gaveen5555** [dotintent/react-native-ble-plx#909]
I recently tried this library and tried to detect my simulated beacon.

1. I am running the latest version of react native.
2. I checked the documentation and issues section, and found similar issues but with no solutions.

This is the application i used to simulate my beacon

https://www.microsoft.com/en-lk/p/beacon-simulator/9nblggh4xvd0?activetab=pivot:overviewtab

## Expected Behavior

Detect my simulated beacon with it's name, UUID etc.

## Current Behavior

It detects the bluetooth devices but only shows id as something like 32:BD:32:6C:E9:C2 but no other data is show.
Ex:

'device', { serviceUUIDs: null,
      isConnectable: null,
      overflowServiceUUIDs: null,
      txPowerLevel: null,
      serviceData: null,
      manufacturerData: 'TEACFCd6h5jcoxKqh9ACQqwTAAOBqZYcxQ==',
      name: null,
      mtu: 23,
      rssi: -47,
      solicitedServiceUUIDs: null,
      localName: null,
      id: '32:BD:32:6C:E9:C2',

But i tried other ble scanners in the playstore and they were able to detect my simulated beacon with uuid i have given to the simulated beacon. When i tried real devices such as bluetooth headphones, then their uuid is shown along with its name when scanned using react-native-ble-plx library.

So to sum it up

simulated beacon data is not detected using react-native-ble-plx library. Real devices are detected correctly.

What could be the reason for this?

**Comment by mciechanowicz** [comment]
Hi,

The discussion already exists in #516 as @usichenko-nikita mentioned, so I am closing this topic.
